lib/matplotlib/animation.py

```diff
@@ -62,8 +63,8 @@
 
 
 def adjusted_figsize(w, h, dpi, n):
-    wnew = int(w * dpi / n) * n / dpi
-    hnew = int(h * dpi / n) * n / dpi
+    wnew = np.round(int(w * dpi / n) * n / dpi)
```

Comment:
Ugh, good point!

Unfortunately putting the rounding in `frame_size` produces a corrupted movie:

http://imgur.com/a/CjftT